src/core/LockupNFTDescriptor.sol

```diff
@@ -7,7 +7,7 @@ import { IERC721Metadata } from "@openzeppelin/contracts/token/ERC721/extensions
 import { Base64 } from "@openzeppelin/contracts/utils/Base64.sol";
 import { Strings } from "@openzeppelin/contracts/utils/Strings.sol";
 import { ILockupNFTDescriptor } from "./interfaces/ILockupNFTDescriptor.sol";
-import { ISablierLockup } from "./interfaces/ISablierLockup.sol";
+import { ISablierLockupBase } from "./interfaces/ISablierLockupBase.sol";
```

Comment:
we can use `ISablierLockup` instead of `ISablierLockupBase`

Reply:
Agree.